src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -389,6 +396,13 @@ export class SavedObjectsService
         }
         this.savedObjectServiceCustomStatus$ = status$;
       },
+      addExtendedSavedObjectsMappings: (mappings: SavedObjectsMappingProperties[]) => {
+        if (!this.extendedSavedObjectsMappings) {
+          this.extendedSavedObjectsMappings = mappings;
+        } else {
+          this.extendedSavedObjectsMappings = [...this.extendedSavedObjectsMappings, ...mappings];
```

Comment:
We should check the existing property so that it won't been overwritten. similar to `src/core/server/saved_objects/saved_objects_type_registry.ts` at line 53

Reply:
Sure, it's needed, I'll add this logic.